x-pack/plugins/maps/public/classes/styles/vector/vector_style.tsx

```diff
@@ -239,11 +241,173 @@ export class VectorStyle implements IVectorStyle {
     );
   }
 
+  async _updateFieldsInDescriptor(
+    nextFields: IField[],
+    styleFieldsHelper: StyleFieldsHelper,
+    originalProperties: VectorStylePropertiesDescriptor,
```

Comment:
Why is `originalProperties` passed in a parameter when it can just be retrieved from the instance? So instead of passing in, just `const originalProperties = this.getRawProperties();` directly in the method.